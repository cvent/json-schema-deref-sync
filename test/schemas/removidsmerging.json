{
  "$id": "removeidsmergeadditionalproperties.json",
  "$ref": "bar.json"
}
