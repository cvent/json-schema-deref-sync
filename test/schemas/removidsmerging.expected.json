{
  "$id": "removeidsmergeadditionalproperties.json",
  "description": "bar property",
  "type": "boolean"
}
